bitnami/mariadb-galera/README.md

```diff
@@ -77,6 +77,7 @@ The following table lists the configurable parameters of the MariaDB Galera char
 | `service.port`                             | MariaDB service port                                                                                                                                                                                                                                                             | `3306`                                                            |
 | `service.clusterIP`                        | Specific cluster IP when service type is cluster IP. Use `None` for headless service                                                                                                                                                                                             | `nil`                                                             |
 | `service.nodePort`                         | Kubernetes Service nodePort                                                                                                                                                                                                                                                      | `nil`                                                             |
+| `service.externalIPs`         | External IP list to use with ClusterIP service type                                                        | `[]`                                                             |
```

Comment:
Could you fix the indentation of this row to be consistent with the rest of the table?

Reply:
Done :)